### DISCUSSION TITLE
Option to skip draft PRs in scheduled reminders

### DISCUSSION THREAD

**Discussion by gmeshkov** [community/community#65162] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

An option to skip draft PRs in `Review requests assigned to you` / `Review requests assigned to your team` would be very useful. Scheduled reminders are used to remind you that you should review a PR. We use them a lot to improve code review times.
Draft PRs are not really ready for merging and not require timely review, so they are of no interest for regular reminders. They can be open for a long time and just clutter the reminder messages and irritate you.

**Comment by rolandtritsch** [comment]
Absolutely yes. We want this. We need this. Thanks for raising/proposing this (again).

**Comment by michaeljpeters** [comment]
We want this too!

**Comment by nkukushkin-pro** [comment]
Yes, this would be very useful to have as an option

**Comment by vbuell** [comment]
There is absolutely no value of distracting people if the work is not done (and that's exactly the reason the `Draft` state exists).
